bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/service/ServiceResponse.java

```diff
@@ -0,0 +1,57 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.http.service;
+
+import org.apache.bookkeeper.http.HttpServer;
+
+/**
+ * A wrapper class that wrap the result from service, which
+ * can then be parsed into http response.
+ */
+public class ServiceResponse {
+    private String body;
+    private HttpServer.StatusCode code = HttpServer.StatusCode.OK;
+
+    public ServiceResponse() {}
+
+    public ServiceResponse(String body, HttpServer.StatusCode code) {
+        this.body = body;
+        this.code = code;
+    }
+
+    public String getBody() {
```

Comment:
I think it would be more future-proof not to return a java.lang.String.
IMHO It would be better to have these methods:
- writeTo(OutputStream method)   (or a getInputStream())
- getContentType()
- getContentLength()

If in the future maybe we will deal with large responses and storing them all in a a single String variable could be a problem